.github/workflows/system-test-notification.yml

```diff
@@ -0,0 +1,32 @@
+name: system-test-notification
+
+on:
+  # This needs to be `pull_request_target` instead of `pull_request`
+  # to be able to write a comment on the pull request.
+  pull_request_target:
+    types:
+      - opened
+
+jobs:
+  notify:
+    # This job runs only if the pull request is from a forked repository.
+    if: github.event.pull_request.head.repo.full_name != github.repository
```

Comment:
why not use `github.event.pull_request.head.repo.fork == true`?

Reply:
https://github.com/elastic/apm-queue/pull/233/commits/7cbd688bd21f851e771b043caeeb49b76d5b7776